docs/docs/content/in-depth/12_computed.en.md

```diff
@@ -4,18 +4,19 @@ title: Computed
 
 ## Computed
 
-Very often the components needs to produce state based on a combination of other state. A very typical example is to show items of a list based on a dynamic limit. The whole list is stored in your state tree, but you only want to show for example the 10 latest. Instead of defining this in the component itself, you can use a **computed**. The signature is very much like connecting state to a component, but instead of returning a user interface description, it returns a value.
+Very often the components needs to produce state based on a combination of other state. A very typical example is to show items of a list based on a dynamic limit. The whole list is stored in your state tree, but you only want to show for example the 10 latest. Instead of defining this in the component itself, you can use **compute**.
 
 ```js
-import {Computed} from 'cerebral'
+import {compute} from 'cerebral'
 import {state} from 'cerebral/tags'
 
-export default Computed({
-  limit: state`items.limit`,
-  items: state`items.list`
-}, ({limit, items}) => {
-  return items.filter((item, index) => index < limit)
-})
+export default compute(
+  state`items.limit`,
+  state`item.list`,
+  (limit, list) => {
```

Comment:
the param name should be `items` instead of `list` to make the example 100% correct 😃